.github/workflows/pull-request.yaml

```diff
@@ -274,3 +274,33 @@ jobs:
       - name: Build
         run: make ci-go-ci-build-linux GOVERSION=${{ matrix.version }}
         timeout-minutes: 30
+
+  # Run PR metadata against Rego policies
+  rego-check-pr:
+    name: Rego PR checks
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Download OPA
+        uses: infracost/setup-opa@v1
```

Comment:
This turned out to be really handy! Perhaps we should wait for the transfer of this into the OPA project to be finished before we use it though?

Reply:
Alternatively, we could use the OPA built in other steps.. which would be kind of a smoke test for the binary I suppose? 🤔

Reply:
Let's use latest for now and see how it goes

Reply:
Agreed - it's great to "dog food" the GH Actions project as well!